stats_tracker/stats_tracker/leaderboards/leaderboards.py

```diff
@@ -149,29 +154,65 @@ def _get_leaderboard_set_id(product_id: str, sub_key: str) -> str:
     return f"_lb:{sub_key}:{product_id}"
 
 
+def _get_users_leaderboard_set_id(user_id: str) -> str:
+    """Return a key for given user's list of leaderboards he's in."""
+    return f"_lb:{user_id}:leaderboard_set"
+
+
+def add_to_users_leaderboard_set(user_id: str, leaderboard_id: str):
+    # Record the leaderboard id in user's lists of leaderboards
+    try:
+        db = get_stats_tracker_db()
+        users_leaderboard_set_id = _get_users_leaderboard_set_id(user_id)
+        db.sadd(users_leaderboard_set_id, leaderboard_id)
+        logging.info(f"Added leaderboard {leaderboard_id} to user {user_id} record")
+    except RedisError as err:
+        logging.error(
```

Comment:
Worth using 'logging.exception' here - python will print out the stack trace for you